cmd/kubeadm/app/cmd/certs.go

```diff
@@ -335,6 +336,14 @@ func getInternalCfg(cfgPath string, kubeconfigPath string, cfg kubeadmapiv1.Clus
 			internalcfg, err := configutil.FetchInitConfigurationFromCluster(client, nil, logPrefix, false, false)
 			if err == nil {
 				fmt.Println() // add empty line to separate the FetchInitConfigurationFromCluster output from the command output
+				// certificate renewal or expiration checking doesn't depend on a running cluster, which means the CertificatesDir
+				// could be set to a value other than the default value or the value fetched from the cluster.
+				// cfg.CertificatesDir could be empty if the default value is set to empty (not true today).
+				klog.V(1).Infof("Overriding the default cert dir with the value from command line flag: %s", cfg.CertificatesDir)
+				if len(cfg.CertificatesDir) != 0 {
```

Comment:
```suggestion
				if len(cfg.CertificatesDir) != 0 {
					klog.V(1).Infof("Overriding the cluster certificate directory with the value from command line flag --%s: %s", options.CertificatesDir, cfg.CertificatesDir)
```

flag name comes from:
https://github.com/kubernetes/kubernetes/blob/0985c476e3db8a5f9532eb4be5cd8adc6f94deef/cmd/kubeadm/app/cmd/options/constant.go#L33

Reply:
thanks!